### PR TITLE
Set individual dates as disabled

### PR DESCRIPTION
**Pull request by jwd2a**
Adds the ability to set individual dates as disabled, in case you're using this and need to show blocked out dates.

### CODE REVIEW

**Comment by wojtekmaj** [comment]
Nice one! Please resolve the `Uncaught TypeError` issue and I'll be happy to merge!

**Comment by jwd2a** [comment]
I'll get that patched up today!

File: src/Tile.jsx [hunk 1]
```diff
@@ -27,7 +28,8 @@ const Tile = ({
     )}
     disabled={
       (minDate && minDateTransform(minDate) > date) ||
-      (maxDate && maxDateTransform(maxDate) < date)
+      (maxDate && maxDateTransform(maxDate) < date) ||
+      tileDisabled({ date, view })
```
Comment (wojtekmaj) [inline src/Tile.jsx]: You should handle it like so:

```js
(tileDisabled && tileDisabled({ date, view }))
```

Comment (jwd2a) [reply src/Tile.jsx]: :thumbsup: Yup, forgot that sanity check

**Comment by wojtekmaj** [comment]
Awesome job! Congrats and thanks for your first contribution :)

**Pull request merged**